### PR TITLE
Create integrations.md

### PR DESCRIPTION
**Pull request by bstuder99**
Responds to https://github.com/kubecost/docs/issues/630 to create a list of supported OpenCost integrations
Relevant integration/config docs are linked where they already exist. We can work on developing specific integration guides for everything else over time

### CODE REVIEW

File: docs/integrations.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 9
+---
+# OpenCost Integrations Support
+Below is a list of all officially-supported platforms for integrations with OpenCost.
+
+## Cloud service providers
+* [AWS](https://github.com/opencost/opencost-website/blob/main/docs/AWS.md)
```
Comment (mattray) [inline docs/integrations.md:8]: Could we change this to https://www.opencost.io/docs/AWS

File: docs/integrations.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* [Azure](https://github.com/opencost/opencost-website/blob/main/docs/azure-opencost.md)
```
Comment (mattray) [inline docs/integrations.md:9]: Could we change this to https://www.opencost.io/docs/azure-opencost

**Review by mattray — CHANGES_REQUESTED** [review]

**Comment by bstuder99** [comment]
I'm pretty sure I grabbed every URL from the frontend, not GH. In certain cases (in Kubecost docs), these links direct you to the platform you are currently browsing. ie, browsing docs in GH will link you to more GH docs, but browsing docs in the frontend links you to other frontend docs.

**Comment by mattray** [comment]
The Vercel preview shows the links going to GitHub, maybe the Kubecost docs work differently when rendered.
https://opencost-website-git-bstuder99-patch-3-kubecost.vercel.app/docs/integrations

**Comment by bstuder99** [comment]
Okay, everything's been manually corrected

**Review by mattray — APPROVED** [review]

**Comment by mattray** [comment]
Thanks!

**Pull request merged**